lib/services/app-files-updater.ts

```diff
@@ -81,6 +79,7 @@ export class AppFilesUpdater {
 				return this.fs.createDirectory(destinationPath);
 			}
 
+			console.log(`Copying ${source} to ${destinationPath}`);
```

Comment:
![meme_console_log](https://user-images.githubusercontent.com/8351653/36141636-16adb4fc-10ae-11e8-9974-49ad75ce5b06.jpg)

Reply:
This is actually a forgotten debug message, will remove